modules/oe_content_event/src/Entity/EventVenue.php

```diff
@@ -0,0 +1,374 @@
+      ->setLabel(t('Capacity'))
+      ->setDescription(t('The capacity of the venue.'))
+      ->setRevisionable(TRUE)
+      ->setTranslatable(TRUE)
+      ->setSettings([
+        'max_length' => 50,
+        'text_processing' => 0,
+      ])
+      ->setDefaultValue('')
+      ->setDisplayOptions('view', [
+        'label' => 'above',
+        'type' => 'string',
+        'weight' => -4,
+      ])
+      ->setDisplayOptions('form', [
+        'type' => 'string_textfield',
+        'weight' => -4,
+      ])
+      ->setDisplayConfigurable('form', TRUE)
+      ->setDisplayConfigurable('view', TRUE)
+      ->setRequired(FALSE);
+
+    $fields['room'] = BaseFieldDefinition::create('string')
+      ->setLabel(t('Room'))
+      ->setDescription(t('The name of the room.'))
+      ->setRevisionable(TRUE)
+      ->setTranslatable(TRUE)
+      ->setSettings([
+        'max_length' => 50,
+        'text_processing' => 0,
+      ])
+      ->setDefaultValue('')
+      ->setDisplayOptions('view', [
+        'label' => 'above',
+        'type' => 'string',
+        'weight' => -4,
+      ])
+      ->setDisplayOptions('form', [
+        'type' => 'string_textfield',
+        'weight' => -4,
+      ])
+      ->setDisplayConfigurable('form', TRUE)
+      ->setDisplayConfigurable('view', TRUE)
+      ->setRequired(FALSE);
+
+    $fields['postal_address'] = BaseFieldDefinition::create('address')
+      ->setLabel(t('Postal address'))
+      ->setDescription(t('The address of the venue.'))
+      ->setRevisionable(TRUE)
+      ->setSettings([
+        'available_countries' => [],
+        'langcode_override' => '',
+        'field_overrides' => [
+          'givenName' => [
+            'override' => 'hidden',
+          ],
+          'additionalName' => [
+            'override' => 'hidden',
+          ],
+          'familyName' => [
+            'override' => 'hidden',
+          ],
+        ],
+        'fields' => [],
+      ])
+      ->setDisplayOptions('view', [
+        'label' => 'above',
+        'type' => 'string',
+        'weight' => -4,
+      ])
+      ->setDisplayOptions('form', [
+        'type' => 'string_textfield',
+        'weight' => -4,
+      ])
+      ->setDisplayConfigurable('form', TRUE)
+      ->setDisplayConfigurable('view', TRUE)
+      ->setRequired(FALSE);
+
+    $fields['status']->setDescription(t('A boolean indicating whether the Event venue is published.'))
+      ->setDisplayOptions('form', [
+        'type' => 'boolean_checkbox',
+        'weight' => -3,
+      ]);
+
+    $fields['created'] = BaseFieldDefinition::create('created')
+      ->setLabel(t('Created'))
+      ->setDescription(t('The time that the entity was created.'));
+
+    $fields['changed'] = BaseFieldDefinition::create('changed')
+      ->setLabel(t('Changed'))
+      ->setDescription(t('The time that the entity was last edited.'));
+
+    $fields['revision_translation_affected'] = BaseFieldDefinition::create('boolean')
```

Comment:
Is this really needed?I think it gets created automatically.

Reply:
I think it is. In node module they have added this by an upgrade path, and from install.